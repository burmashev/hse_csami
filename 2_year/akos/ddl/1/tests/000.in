2022/03/11
2022/03/12
