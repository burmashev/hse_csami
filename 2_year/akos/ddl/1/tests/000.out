2022/03/24 23:59:59
ERROR
